scheduler/src/main/java/io/camunda/zeebe/scheduler/ActorThread.java

```diff
@@ -98,17 +98,19 @@ private void doWork() {
   private void executeCurrentTask() {
     final var properties = currentTask.getActor().getContext();
     MDC.setContextMap(properties);
+    MDC.put("actor-scheduler", actorThreadGroup.getSchedulerName());
```

Comment:
This happens now on each loop iteration, could we somehow set this only once on thread creation :thinking:

Reply:
Unfortunately this was required because `MDC.setContextMap` clears the context completely =/

Reply:
We could put each property from the MDC instead, but then we wouldn't be clearing it. Clearing it means we can't set it only once per thread unfortunately :(